### PR TITLE
Migrate from REACT to BLAZE.

### PR DESCRIPTION
**Pull request by kuksik**

### CODE REVIEW

File: imports/ui/components/Goal.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      		:
+        		this.props.goal.title 
+      		}
+          &nbsp;
+      		({ this.props.goal.progress || 0 }%)
+  		  </div>
+  		</a>
+		)
+	}
+}
```
Comment (epaminond) [inline imports/ui/components/Goal.jsx:52]: @kuksik, could you convert tabs into spaces before we merge this PR?

File: imports/ui/components/Navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import ReactDOM from 'react-dom';
+import { Template } from 'meteor/templating';
+import { Blaze } from 'meteor/blaze';
+import ModalAddStage from './ModalAddStage.jsx';
+import ModalLoggedAlert from './ModalLoggedAlert.jsx';
+import I18n from 'meteor/timoruetten:react-i18n';
+import { Stages } from '/imports/api/stages/stages.js';
+
+
+
```
Comment (epaminond) [inline imports/ui/components/Navigation.jsx:10]: Please, use one delimiter-line not three.

**Comment by epaminond** [comment]
Please add node-modules to gitignore. It shouldn't get to the repository.

**Comment by epaminond** [comment]
@kuksik, I've removed node_modules myself and added proper eslint rules. To run eslint checks you need to execute `npm run lint`. It currently shows multiple (2435) errors. Could you fix them before we merge?

**Comment by kuksik** [comment]
@epaminond  I've fixed most errors and added npm js-yaml package.

File: client/main.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<head>
+  <title>Pelorus</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta desiption='Pelorus is a simple yet highly-effective planning tool that
+                   helps reaching goals in a way of successive approximations
+                   (based on Dynamic programming).'>
+</head>
```
Comment (kuksik) [inline client/main.html:9]: Here I added a title and meta with description.

Comment (epaminond) [inline client/main.html:9]: Greate job!

**Pull request merged**